web/crux-ui/src/components/products/versions/deployments/instances/instance-log-terminal.tsx

```diff
@@ -0,0 +1,78 @@
+import { ContainerLogMessage } from '@app/models'
```

Comment:
is this terminal component any way different than the container one?

Reply:
They only differ in the message formatter. I just realized that I could move that to a prop and use the same component for both.